### PR TITLE
expense

### PR DESCRIPTION
**Pull request by Sivakumar-21**

### CODE REVIEW

File: app/Http/Controllers/ClientsController.php [hunk 1]
```diff
@@ -115,6 +120,8 @@ public function contactedit(UpdateCustomerContactRequest $request)
          $customer_contact->email = $request->email;
          $customer_contact->client_id = $request->client_id;
          $customer_contact->mobile = $request->mobile;
+         $customer_contact->is_customer = $request->input('is_customer1', false);
```
Comment (muni2explore) [inline app/Http/Controllers/ClientsController.php:123]: check and update the name

File: database/migrations/2023_04_21_094740_create_invoice_description_table.php
Comment (muni2explore) [inline database/migrations/2023_04_21_094740_create_invoice_description_table.php:1]: change name to invoice_details

**Review by muni2explore — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/ClientsController.php (hunk 1 above)
Comment (Sivakumar-21) [reply app/Http/Controllers/ClientsController.php:123]: correction changed

File: database/migrations/2023_04_21_094740_create_invoice_description_table.php
Comment (Sivakumar-21) [reply database/migrations/2023_04_21_094740_create_invoice_description_table.php:1]: changed

**Pull request merged**